boards/nordic/nrf54h20dk/Kconfig.defconfig

```diff
@@ -15,3 +15,6 @@ config BT_CTLR
 	default y if BT
 
 endif # BOARD_NRF54H20DK_NRF54H20_CPURAD
+
+config SPI_DW_HSSI
+	default y if SPI_DW
```

Comment:
is this really a board-level defconfig?

Reply:
Right, that was the wrong one. Moved to soc defconfig.